src/lib.rs

```diff
@@ -311,6 +323,7 @@ pub struct Package {
     /// Targets provided by the crate (lib, bin, example, test, ...)
     pub targets: Vec<Target>,
     /// Features provided by the crate, mapped to the features required by that feature.
+    #[serde(serialize_with = "sorted_map")]
     pub features: HashMap<String, Vec<String>>,
```

Comment:
Personally I would prefer if the type of this field was simply changes, but **I'm not a maintainer**.

Reply:
I agree. We can do this when we do the next major version bump. I'll open an issue for it